app/models/manageiq/providers/vmware/inventory/collector/network_manager.rb

```diff
@@ -1,177 +1,2 @@
 class ManageIQ::Providers::Vmware::Inventory::Collector::NetworkManager < ManageIQ::Providers::Vmware::Inventory::Collector
-  VappNetwork = Struct.new(:id, :name, :type, :is_shared, :gateway, :dns1, :dns2, :netmask, :enabled, :dhcp_enabled, :orchestration_stack)
-
-  def initialize(_manager, _target)
-    super
-
-    initialize_network_inventory_sources
-  end
-
-  def initialize_network_inventory_sources
-    @vdc_networks         = []
-    @vdc_networks_idx     = {}
-    @vapp_networks        = []
-    @network_ports        = []
-    @routers              = []
-    @network_name_mapping = {}
-  end
-
-  def orgs
-    return @orgs if @orgs.any?
-    @orgs = connection.organizations || []
-  end
-
-  def org
-    orgs.first
-  end
-
-  def vdc_networks
-    return @vdc_networks if @vdc_networks.any?
-    @vdc_networks = org.networks || []
-    @vdc_networks_idx = @vdc_networks.index_by(&:id)
-
-    @vdc_networks
-  end
-
-  def routers
-    return @routers if @routers.any?
-
-    # Routers can't be independently collected, we have to get vapp_networks first
-    vapp_networks
-
-    @routers
-  end
-
-  def vapp_networks
-    return @vapp_networks if @vapp_networks.any?
-    @vapp_networks = manager.orchestration_stacks.each_with_object([]) do |stack, res|
-      fetch_network_configurations_for_vapp(stack.ems_ref).map do |net_conf|
-        # 'none' is special network placeholder that we must ignore
-        next if net_conf[:networkName] == 'none'
-
-        network_id = network_id_from_links(net_conf)
-        if (vdc_net = corresponding_vdc_network(net_conf, @vdc_networks_idx))
-          memorize_network_name_mapping(stack.ems_ref, vdc_net.name, vdc_net.id)
-        else
-          memorize_network_name_mapping(stack.ems_ref, net_conf[:networkName], network_id)
-          res << build_vapp_network(stack, network_id, net_conf)
-
-          # routers connecting vApp networks to VDC networks
-          if (parent_net = parent_vdc_network(net_conf, @vdc_networks_idx))
-            @routers << {
-              :net_conf   => net_conf,
-              :network_id => network_id,
-              :parent_net => parent_net
-            }
-          end
-        end
-      end
-    end
-  end
-
-  def network_ports
-    return @network_ports if @network_ports.any?
-    @network_ports = manager.vms.each_with_object([]) do |vm, res|
```

Comment:
Now that this is all in one refresh we can reference the `@vms` collected inventory here and not rely on vms saved in the database